tests/test_polars_delta.py

```diff
@@ -101,7 +101,16 @@ def append_asset() -> pl.DataFrame:
 
     pl_testing.assert_frame_equal(pl.concat([df, df]), pl.read_delta(saved_path))
 
-
+def test_polars_delta_io_manager_merge(polars_delta_io_manager: PolarsDeltaIOManager):
```

Comment:
Please add a test for the new `merge` functionality. You can take a look at other tests for inspiration. Let me know if you need any help with this